embedded/sql/sql_tx.go

```diff
@@ -24,6 +24,18 @@ import (
 	"github.com/codenotary/immudb/embedded/store"
 )
 
+func NewSQLTx(engine *Engine, opts *TxOptions, tx *store.OngoingTx, catalog *Catalog, currentDB *Database) *SQLTx {
```

Comment:
probably not being used

engine has a method to create transactions, is this method created with a different logic?

Reply:
not required, was dead code from previous implementation